src/pandamonium/pandamon.py

```diff
@@ -243,12 +242,12 @@ def getstatus(task, args):
             outputString = fmt_string.format(
                 s=status_color,
                 t=task['taskname'],
-                i=task['reqid'],
+                i=task['jeditaskid'],
```

Comment:
Seems reasonable here. The only real use for this number is that it's the key to kill the task.